### PR TITLE
chore(card): updated card demo select to be plain, fixed alignment

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/4553
fixes https://github.com/patternfly/patternfly/issues/4621

@mcarrano I just added custom CSS to change the card action/title position so they're aligned to baseline.

### CODE REVIEW

**Review by mcarrano — APPROVED** [review]
I think that these look much better. Thanks @mcoker

**Comment by srambach** [comment]
Looks good. The workspace tweak should be documented in a note near the examples using it for someone who would like to replicate this to know that they need to use a layout or custom CSS to make this line up properly.

**Comment by mcoker** [comment]
@srambach thanks, updated!

**Review by srambach — APPROVED** [review]
I have to admit I'm struggling with the wording of the note, as it implies that you've set `vertical-align:baseline;` somewhere rather than tweaking the padding. And since this is in workspace CSS, a reader will need to do some work to find it in the inspector. I know it's a minor thing, but it's bugging me that workspace code is being used to fix the demo as opposed to fixing the framework around the demo or showing something invisible like we do in the layouts. That said, I don't have a better solution so I'll approve. 😕  🤷‍♀️

**Comment by srambach** [comment]
@mcoker Yeah - I thought about inline styles..if you can do that I think that would a good solution.

**Comment by mcoker** [comment]
@srambach updated

File: src/patternfly/demos/Card/examples/Card.md [hunk 1]
```diff
@@ -959,8 +963,8 @@ import './Card.css'
           Workload
         </span>
       {{/l-flex}}
-      {{#> card-actions}}
-        {{#> select id=(concat card--id '-select-dropdown') select-menu--modifier="pf-m-align-right"}}
+      {{#> card-actions card-actions--modifier="pf-m-no-offset" card-actions--attribute='style="padding-top: 3px;"'}}
```
Comment (srambach) [inline src/patternfly/demos/Card/examples/Card.md:966]: Was this supposed to be 1 px?

**Review by srambach — APPROVED** [review]
Awesome, I like this much better. Just checking on one adjustment. Thanks!

File: src/patternfly/demos/Card/examples/Card.md (hunk 1 above)
Comment (mcoker) [reply src/patternfly/demos/Card/examples/Card.md:966]: doh!

**Pull request merged**